learning_journal_basic/views/views.py

```diff
@@ -0,0 +1,73 @@
+"""
+Views controller for Pyramid Learning Journal application
+"""
+
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPNotFound
+from pyramid.httpexceptions import HTTPFound
+import pytz
+from datetime import datetime
+
+from ..models import PLJ_Article
+
+@view_config(route_name='home', renderer='../templates/list.jinja2')
+def list_view(request):
+    """
+    This will return the index page for the Learning Journal
+    """
+    articles = request.dbsession.query(PLJ_Article).order_by(PLJ_Article.date_created.desc())
+    article = articles.all()
```

Comment:
Line 18 is probably better served as 
```python
query = request.dbsession.query(PLJ_Article).order_by(PLJ_Article.date_created.desc())
```
That way, line 19 could be
```python
articles = query.all()
```
And you can have better, more descriptive, less confusing variable names.